sdk/src/wallet/error.rs

```diff
@@ -112,6 +109,12 @@ pub enum Error {
     /// Transaction not found
     #[error("transaction {0} not found")]
     TransactionNotFound(TransactionId),
+    // TODO more precise error
```

Comment:
TODO (want to create an issue?)

Reply:
It's irrelevant to this PR and there is already an issue to handle TODOs

Reply:
okidoki